app/views/dashboard/_foreman_virt_who_configs_status_widget.html.erb

```diff
@@ -0,0 +1,59 @@
+<% scope = ForemanVirtWhoConfigure::Config.for_organization(Organization.current) %>
+<% scope = scope.authorized(:view_virt_who_config) %>
+<% expired = scope.expired.order('expires_at DESC') %>
+
+<h4 class="header ca"><%= _('Virt-who Configurations Statistics') %></h4>
+<table class="table table-striped ellipsis">
+  <thead>
+  <tr>
+    <th><%= _("Status") %></th>
+    <th><%= _("Count") %></th>
+  </tr>
+  </thead>
+  <tbody>
+  <tr>
+    <td><%= link_to _("Configurations Without Single Report"), foreman_virt_who_configure_configs_path(:search => "status = unknown") %></td>
+    <td><%= scope.where(:last_report_at => nil).count %></td>
+  </tr>
+  <tr>
+    <td><%= link_to _("Expired Configurations"), foreman_virt_who_configure_configs_path(:search => "status = error") %></td>
```

Comment:
`"Out of sync"` is probably more suitable. Can you rename it, please? The scopes and methods should be probably renamed too. At least the scoped search field names that leak into UI.